ui/hooks/useCurrencyDisplay.js

```diff
@@ -172,6 +174,10 @@ export function useCurrencyDisplay(
       });
     }
 
+    if (isAggregatedFiatOverviewBalance) {
+      return formatCurrency(inputValue, currency);
+    }
```

Comment:
Can you explain why this is a special case?  It's a little messy for this generic hook to need knowledge about a specific caller.

Reply:
The input value we get from the hook `useAccountTotalFiatBalance` is already in decimal format  so i added this check in this hook so it only applies `formatCurrency` to the value.
In the other cases, `inputValue` we get other displays is in hex; which required `formatEthCurrencyDisplay`